File: src/main/graphql/io/github/graphql/resolver/PullRequestReviewContributionsByRepositoryResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for PullRequestReviewContributionsByRepository
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface PullRequestReviewContributionsByRepositoryResolver {

    @Throws(Exception::class)
    fun contributions(pullRequestReviewContributionsByRepositoryTO: PullRequestReviewContributionsByRepositoryTO, after: String?, before: String?, first: Int?, last: Int?, orderBy: ContributionOrderTO?): CreatedPullRequestReviewContributionConnectionTO

}

File: src/main/scala-graphql/io/github/graphql/scala/resolver/PullRequestReviewContributionsByRepositoryResolver.scala
package io.github.graphql.scala.resolver

import io.github.graphql.scala.model._

/**
 * Resolver for PullRequestReviewContributionsByRepository
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
trait PullRequestReviewContributionsByRepositoryResolver {

    @javax.validation.constraints.NotNull
    @throws[Exception]
    def contributions(pullRequestReviewContributionsByRepositoryTO: PullRequestReviewContributionsByRepositoryTO, after: String, before: String, first: Option[Int], last: Option[Int], orderBy: ContributionOrderTO): CreatedPullRequestReviewContributionConnectionTO

}
